Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules_README.md

```diff
@@ -1,11 +1,12 @@
-This playbook returns "RemediationAction" options based on the return from the Remediation Path Rules API, or defaults to data collection task options from the "Cortex ADM - Decision" sub-playbook.
+This playbook returns "RemediationAction" options based return from Remediation Path Rules API, or defaults to data collection task options from "Cortex ADM - Decision" subplaybook.
```

Comment:
yes, I agree and ADM vs ASM.  wondering if the README was changed but not the playbook description that populates it.

Reply:
after looking at this, I think the change in the README is preferred because it was Shirley's wording: https://github.com/demisto/content/pull/24380#discussion_r1101167240.

Updating playbook YML to have the same thing

Reply:
[John's suggestions](https://github.com/demisto/content/pull/27795/commits/2e73af20c83edcef545f912c1964ef64fb572005)